lib/ansible/modules/commands/shell.py

```diff
@@ -87,6 +87,8 @@
     do not include evil things like semicolons.
   - An alternative to using inline shell scripts with this module is to use
     the M(script) module possibly together with the M(template) module.
+  - For Windows targets, use the M(win_shell) module instead.
```

Comment:
We have been removing these from the notes, because we have this line at the very top in the description, and a reference to the **win_shell** module in the **seealso:** section. So three times is one to many.

We have been removing those from the notes for all Windows modules, and vice versa.